webknossos/webknossos/geometry/vec3_int.py

```diff
@@ -112,6 +112,8 @@ def from_vec_or_int(vec_or_int: Union["Vec3IntLike", int]) -> "Vec3Int":
     def from_str(string: str) -> "Vec3Int":
         if re.match(r"\(\d+,\d+,\d+\)", string):
             return Vec3Int(tuple(map(int, re.findall(r"\d+", string))))
+        elif re.match(r"^\d+,\d+,\d+$", string):
```

Comment:
Vielleicht ein kurzer Kommentar, was die Regex macht. Speziell im Vergleich zu bestehenden regex.

Reply:
Sure, I added a comment for both regex and fixed an issue with the first regex. The first regex evaluated to true even if the correct pattern, e.g., `(10,20,30)` is only a substring of `string`. So the map could deliver more than three integers. This should be fixed now.